comp/process/agent/agentimpl/agent.go

```diff
@@ -0,0 +1,115 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package agentimpl implements a component for the process agent.
+package agentimpl
+
+import (
+	"context"
+
+	"github.com/opentracing/opentracing-go/log"
+	"go.uber.org/fx"
+
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	logComponent "github.com/DataDog/datadog-agent/comp/core/log"
+	"github.com/DataDog/datadog-agent/comp/process/agent"
+	"github.com/DataDog/datadog-agent/comp/process/runner"
+	"github.com/DataDog/datadog-agent/comp/process/types"
+	"github.com/DataDog/datadog-agent/pkg/process/checks"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+	"github.com/DataDog/datadog-agent/pkg/util/optional"
+)
+
+const (
+	agent6DisabledMessage = `process-agent not enabled.
```

Comment:
is there a reason this is specifically called `agent6`?

Reply:
This was ported from https://github.com/DataDog/datadog-agent/blob/a958bf57751fb97a8c4777c957053f040f798a22/cmd/process-agent/command/main_common.go#L48-L56

I can rename the const as it's version agnostic.